docs/providers/aws/guide/variables.md

```diff
@@ -657,3 +660,15 @@ Resources:
         - Ref: 'AWS::AccountId'
         - 'log-group:/aws/lambda/*:*:*'
 ```
+
+## Casting string variables values to boolean
+
+In some cases, a parameter expect a `true` or `false` value. If you are using a variable to define the value, it may return as a string (e.g. when using SSM variables) and thus return a `"true"` or `"false"` value.
+
+To ensure a boolean value is returned, cast the variable value. For example:
+
+```yml
+provider:
+  tracing:
+    apiGateway: ${toBool:${ssm:API_GW_DEBUG_ENABLED}}
```

Comment:
I can change this to `${toBool(value)}` if that is preferred over `${toBool:value}`.

Reply:
I think it's important to name it as `strToBool` or `stringToBool`.. as otherwise `toBool` suggests language native boolean convertion (which can be assumed as an alias for `Boolean(value)`, when it's not what it does.

Here we're doing something custom (technically it's _boolean string to boolean_ coercion, and not _value to boolean_). So even more accurate name would be `boolStrToBool` but that might be too verbose.

@pmuens thoughts?

Reply:
I see your point. Using `to` in the name will imply casting. What if we use `as` in the name to imply translation. For example, what if we use `strAsBool` or `readStringAsBoolean`?

Reply:
> Using to in the name will imply casting

It is about casting, so `to` sounds fine to me.. Still it should not imply that we simply cast to _boolean_ per language rules (as that's already done internally, no need to use decorator for that).

It's a custom casting (where we cast a boolean string representation to it's boolean counterpart), and it'll be nice if name somehow suggest that.

I think `strToBool` is probably best

Reply:
I made the changes.